README.md

```diff
@@ -7,36 +7,84 @@
 
 ## Built With
 
-- Major languages
+- React
 - Frameworks
 - Technologies used
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
+# Getting Started with Create React App
 
-## Getting Started
```

Comment:
- [ ] [optional] Kindly remove the unnecessary info in your getting started section like how to make a progressive web app.

Reply:
@danmainah Thanks for the feedback, I changed the Readme file back to the Microverse template.